src/controls/extras/treeitem.coffee

```diff
@@ -34,16 +34,25 @@ class TreeItem #extends Base
   handle_relevant_or_required: ->
     tree_div = @tree.tree_root
     current_node = tree_div.find("[node_value = '" + @value + "']")
-
-    current_node.attr('relevant', @node_relevant())
+    current_node.attr('item-required', @node_required())
```

Comment:
For the tree control specifically, I'd make it an ECHO issue. I think it's sufficient to provide publicly-available links to the ECHO Forms spec, schema, and links to documents about extensions (even if they're PDF). I've implemented ECHO Forms twice now, and both times I went off the ECHO Forms spec document. It's really very good. I don't think we need to redo it.

Can we just require people requesting extensions to write up the detailed behavior in the style of the original spec document and store those somewhere we can link to (earthdata.nasa.gov?).